bin/rialto-parachain/runtime/Cargo.toml

```diff
@@ -11,9 +11,9 @@ substrate-wasm-builder = { git = "https://github.com/paritytech/substrate", bran
 
 [dependencies]
 codec = { package = 'parity-scale-codec', version = '3.1.5', default-features = false, features = ['derive']}
+hex-literal = "0.3"
```

Comment:
Is it really required - can't see any added usages of this crate?

Reply:
older code doesnt compile here for `#[cfg(feature = "runtime-benchmarks")]`
https://github.com/paritytech/parity-bridges-common/blob/master/bin/rialto-parachain/runtime/src/lib.rs#L790-L798